enhancements/update/synchronized-upgrades.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Synchronized Upgrades Between Clusters
+authors:
+  - "@danwinship"
+reviewers:
+  - TBD
```

Comment:
@sdodson and someone from the OTA team like @LalatenduMohanty or @wking should be included as reviewers